pkgs/by-name/mi/misc-gitology/package.nix

```
@@ -0,0 +1,45 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  perl,
+  python3,
+}:
+stdenv.mkDerivation {
+  pname = "misc-gitology";
+  version = "unstable-2024-08-26";
```

```suggestion
  version = "0-unstable-2024-08-26";
```
(new policy to always start version with a digit)